extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -108,15 +119,53 @@ export class AmpAudio extends AMP.BaseElement {
   pauseCallback() {
     if (this.audio_) {
       this.audio_.pause();
+      if (getMode().test) {
+        this.isPlaying_ = false;
+      }
+    }
+  }
+
+  pause() {
+    const storyEl = closestByTag(this.element, 'AMP-STORY');
+    if (this.audio_ && !storyEl) {
+      this.audio_.pause();
+      if (getMode().test) {
+        this.isPlaying_ = false;
+      }
     }
   }
 
+  play() {
+    const storyEl = closestByTag(this.element, 'AMP-STORY');
+    if (this.audio_ && !storyEl) {
```

Comment:
This should be storyEl != parentElement?

Reply:
Also do I care about `amp-story` or `amp-story-page`?

Reply:
You can short-circuit more quickly by checking for amp-story-page, but I suggest leaving `amp-story` just for the semantics. I also suggest extracting the check into a separate method and short-circuit clause there. 

A user warning would be nice as well.

```js
if (!this.audio_) {
  return;
} 
if (this.isStoryDescendant_()) {
   user().warn(TAG, '<amp-story> elements do not support actions');
   return;
}
this.audio.play_();
this.setPlayingStateForTesting_(true);
```

Reply:
Done in PR #15222